src/test/regress/pg_regress_multi.pl

```diff
@@ -487,6 +487,7 @@ sub generate_hba
 push(@pgOptions, "citus.enable_manual_changes_to_shards=on");
 push(@pgOptions, "citus.allow_unsafe_locks_from_workers=on");
 push(@pgOptions, "citus.stat_statements_track = 'all'");
+push(@pgOptions, "citus.stats_tenants_limit = 2");
```

Comment:
Any reason to have this change globally instead of only in the test that we care about it?

Reply:
ah yeah makes sense